pages.de/common/git-apply.md

```diff
@@ -14,3 +14,15 @@
 - Integriere eine externe Patch-Datei:
 
 `curl {{https://beispiel.de/datei.patch}} | git apply`
+
+- Gib diffstat des Inputs aus und integriere die Patch-Datei:
+
+`git apply --stat --apply {{pfad/zu/datei}}`
+
+- Integriere eine Patch-Datei in umgekehrter Reihenfolge:
+
+`git apply --reverse {{pfad/zu/datei}}`
+
+- Speichere das Ergebnis einer Patch-Datei im Index/Stage, ohne den Arbeitsbaum zu verändern:
```

Comment:
```suggestion
- Speichere das Ergebnis einer Patch-Datei im Index/in der Stage, ohne den Arbeitsbaum zu verändern:
```

Reply:
Is there a rule to know which gender an English word has in German?
Also, if I change it here, I have to change it in every German `git-` pages.

Reply:
Well, I suppose the German translation would give you the grammatical gender ‒ e.g. "stage" would be translatable by "Bühne" or "Ebene" in German, both being female.

Reply:
Makes sense. How about if we remove `Stage` completely?

Reply:
I don't know enough about Git terminology yet to say whether that's fine to do. However, if Index and Stage are the same concept, then removing it should be safe.

Reply:
As far as I know, they are. @bl-ue any thoughts about this?

Reply:
Yes, they are. To "stage" a file (with `git add`) is the same as "adding the file to the index."

Reply:
Then I'll remove `Stage`, since the english pages only use `Index`